File: go/raindrops/raindrops.go
package raindrops

import (
	"strconv"
)

var (
	numbers = []int{3, 5, 7}
	words   = []string{"Pling", "Plang", "Plong"}
)

func Convert(number int) (result string) {
	for i, divisor := range numbers {
		if number%divisor == 0 {
			result += words[i]
		}
	}

	if result == "" {
		result = strconv.Itoa(number)
	}

	return result
}

File: julia/raindrops/raindrops.jl
const words = ["Pling", "Plang", "Plong"]

function raindrops(number)
    result = ""
    for (i, word) in enumerate(words)
        if number % (2 * i + 1) ≡ 0
            result *= word
        end
    end
    isempty(result) ? string(number) : result
end
